JS/test/airport.spec.js

```diff
@@ -66,30 +66,28 @@ describe('My Test', () => {
             }
         }
         assert.isTrue(nextPlaneMaxLoadCapacityIsHigherThanCurrent);
-    })
+    });
 
-    it('testHasAtLeastOneBomberInMilitaryPlanes', () => {
+    it('List of Bomber planes In Military Planes', () => {
         let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes ();
+        let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes();
         let flag = false;
         for (let militaryPlane of bomberMilitaryPlanes) {
-            if (militaryPlane.getMilitaryType() === MilitaryType.BOMBER) {
+            if (militaryPlane.getMilitaryType() == MilitaryType.Bomber) {
                 flag = true;
             }
             else{
-                assert.fail("Test failed!");
+                assert.fail("No Bomber plains exist!");
             }
         }
+    });
 
-        // if not failed;
-    })
-
-    it('should check that experimentsl planes has classification level higher than unclassified', () => {
+    it('List of experimental planes with classification level higher than unclassified', () => {
         let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getExperimentalPlanes ();
+        let bomberMilitaryPlanes  = airport.getExperimentalPlanes();
         let hasUnclassifiedPlanes  = false;
-        for (let experimentalPlane  of bomberMilitaryPlanes) {
-            if (experimentalPlane.classificationLevel === ClassificationLevel.UNCLASSIFIED) {
+        for (let experimentalPlane of bomberMilitaryPlanes) {
+            if (experimentalPlane.classificationLevel == ClassificationLevel.UNCLASSIFIED) {
```

Comment:
Please correct this code. There is an additional logic implemented. It can be written shorter without any `flag` variable.

Reply:
Done